.github/workflows/session_0.yaml

```diff
@@ -22,10 +22,12 @@ jobs:
         working-directory: session_0
         run: ruby fizz_buzz_test.rb
 
-      - name: Test Word Processing
+      - name: Test Word Processing            
```

Comment:
Instead of a single job that carries on, no matter what - we could split this into four different jobs:

```yaml
jobs:
  setup:
   # check out code
   # install ruby

  fizz_buzz:
    needs: setup
    # test fizz_buzz
  
  word_processing:
    needs: setup
    # test word processing

  ...
```

I had little prior experience with Github Actions when I initially wrote the workflows but this seems to be a better way as all three tests would run in parallel.

https://docs.github.com/en/free-pro-team@latest/actions/learn-github-actions/managing-complex-workflows#creating-dependent-jobs

Also, to help out in identifying the checks - you can rename the workflow from 'Session 0' to 'Session 0 - Learning Ruby'.